src/main/java/hudson/plugins/sshslaves/SSHConnector.java

```diff
@@ -247,6 +259,10 @@ public Integer getRetryWaitTime() {
         return retryWaitTime;
     }
 
+    public Boolean getTcpNoDelay() {
```

Comment:
if we use boolean primitive with default value we do not need this logic, but we can keep the getter

```suggestion
    public boolean getTcpNoDelay() {
```

Reply:
forget it